packages/python-sdk/e2b/sandbox/sandbox_api.py

```diff
@@ -22,9 +22,25 @@ class SandboxInfo:
     end_at: datetime
     """Sandbox expiration date."""
     envd_access_token: Optional[str]
-    """EnvD access token."""
+    """Envd access token."""
     envd_version: Optional[str]
-    """EnvD version."""
+    """Envd version."""
+
+@dataclass
+class ListedSandbox:
```

Comment:
in JS there's also state, CPU and Memory

Reply:
done